src/engraving/rendering/stable/chordlayout.cpp

```diff
@@ -193,7 +193,7 @@ void ChordLayout::layoutPitched(Chord* item, LayoutContext& ctx)
             double arpeggioAccidentalDistance = ctx.conf().styleMM(Sid::ArpeggioAccidentalDistance) * mag_;
             double accidentalDistance = ctx.conf().styleMM(Sid::accidentalDistance) * mag_;
             gapSize = arpeggioAccidentalDistance - accidentalDistance;
-            gapSize -= item->arpeggio()->insetDistance(chordAccidentals, mag_);
+//            gapSize -= ArpeggioLayout::insetDistance(chordAccidentals, mag_, item);
```

Comment:
Why was this line commented out?

Reply:
I moved the layout methods from the Arpeggio class to ArpeggioLayout in rendering/dev, so `item->arpeggio()->insetDistance` no longer exists.  Should I do the same in rendering/stable?

Reply:
Thanks for the explanation. No, we should not change this class. Let's keep your comment